README.md

```diff
@@ -23,29 +24,24 @@ Zivid Python is the official Python package for Zivid 3D cameras. Read more abou
 
 *Windows users also needs to make sure that the Zivid SDK installation folder is in system `PATH` before using the package, not only the terminal PATH variable. The default install location that should be added to system `PATH` is `C:\Program Files\Zivid\bin`.*
```

Comment:
Is it a possibility to copy the required DLLs from C:\Program Files\Zivid\bin to the location where the pyd file is located to avoid having to modify PATH?

Reply:
Not that I am aware of. When and how do you propose to do it?